### PR TITLE
Shanec7/fixing bug 39

### PR DESCRIPTION
**Pull request by shanec7**
### Description

 Fixed issue 39

### Testing Methodologies

 No tests done

### CODE REVIEW

**Comment by RafaePasha** [comment]
Shane,
This code does not change the color. Please fixe and push again

**Review by RafaePasha — CHANGES_REQUESTED** [review]
Does not work

File: calculator.js [hunk 1]
```diff
@@ -430,7 +430,7 @@ $( document ).ready(function() {
     //   - Change "theme name" to a catchy name (can be your name)
     //   - Change the colors blue and red with colors of your choice.
     //   - Uncomment the call to addOption (remove the `//`)
-    // addOption("theme name", "linear-gradient(blue, red)", "red", "red");
+    addOption("poopoopeepee", "linear-gradient(yellow, purple)", "blue", "black");
```
Comment (etcadinfinitum) [inline calculator.js]: LMAO this theme name is :fire:

**Review by etcadinfinitum — COMMENTED** [review]
Looks fine, will wait for someone else to approve.

Good job filling in the description!

Pro Tip: under "Related Issues", you can [use GitHub keywords to automatically close issues](https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords) if you like; this can be done in the PR description or any of the commits on the feature branch (title or body).

**Review by Erica-Fer — DISMISSED** [review]
Lgtm

**Review by etcadinfinitum — APPROVED** [review]
:tada:

**Pull request merged**